### PR TITLE
add cli

### PR DESCRIPTION
**Pull request by royalpinto**

### CODE REVIEW

**Comment by s-r-x** [comment]
@royalpinto, n1! i'll check that asap

File: packages/cli/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "author": "Ilya Strus <strax1990@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/s-r-x/bull-monitor.git",
+    "directory": "packages/cli"
+  },
+  "license": "MIT",
+  "types": "dist/index.d.ts",
+  "bin": {
+    "semver": "dist/index.js"
```
Comment (s-r-x) [inline packages/cli/package.json:17]: Is there any reason why the command is called semver?

File: packages/cli/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# @bull-monitor/cli
+
+Command line interface fpr adapter for [bull-monitor](https://github.com/s-r-x/bull-monitor)
+
+## Usage
+
+```sh
+npm i @bull-monitor/cli
```
Comment (s-r-x) [inline packages/cli/README.md:8]: some usage example would be awesome, e.g. 
```sh
cmd-name -q my-queue
```
or just simply paste  commander --help output, but only with the command name

**Review by s-r-x — CHANGES_REQUESTED** [review]

File: packages/cli/package.json (hunk 1 above)
Comment (royalpinto) [reply packages/cli/package.json]: @s-r-x it was typo. corrected it.

File: packages/cli/README.md (hunk 2 above)
Comment (royalpinto) [reply packages/cli/README.md]: Makes sense, updated it.

**Pull request merged**